com.agatanawrot.karambola.yml

```diff
@@ -0,0 +1,42 @@
+app-id: com.agatanawrot.karambola
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: com.agatanawrot.karambola
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+modules:
+  - name: karambola
+    buildsystem: simple
+
+    sources:
+      - type: archive
+        url: https://holypangolin.com/stuff/karambola/1.5.90/karambola-1.5.90-linux-amd64.tar.gz
+        sha256: ae811fb76fb94ee140183e05a9589c65ea3e3e38a4cfb9d9a553f885333375d3
+        only-arches:
+          - x86_64
+      - type: archive
+        url: https://holypangolin.com/stuff/karambola/1.5.90/karambola-1.5.90-linux-arm64.tar.gz
+        sha256: 163ce2a25e76c851e4fd7d8cbafb9c96a1236f7a4424a79218a5d64ed1dbb776
+        only-arches:
+          - aarch64
+      - type: script
+        dest-filename: com.agatanawrot.karambola
+        commands:
+          - zypak-wrapper.sh /app/bin/karambola "$@"
+
+    build-commands:
+      - install -Dm644 -t /app/share/metainfo resources/app/com.agatanawrot.karambola.metainfo.xml
+      - install -Dm644 -t /app/share/applications resources/app/com.agatanawrot.karambola.desktop
+      - install -Dm644 resources/app/karambola-64.png /app/share/icons/hicolor/64x64/apps/com.agatanawrot.karambola.png
+      - install -Dm644 resources/app/karambola-128.png /app/share/icons/hicolor/128x128/apps/com.agatanawrot.karambola.png
+      - install -Dm644 resources/app/karambola-256.png /app/share/icons/hicolor/256x256/apps/com.agatanawrot.karambola.png
+      - install -Dm644 resources/app/karambola.png /app/share/icons/hicolor/512x512/apps/com.agatanawrot.karambola.png
+      - mv * /app/bin
```

Comment:
are you sure this is right?

Reply:
nevermind. it looks fine. Just unusual.

Reply:
You mean /app/bin? Since nothing gets exported from there it should be right, but I can move it to separate directory anyway.